action.yml

```
@@ -12,6 +12,9 @@ inputs:
     description: "Commit and push changes"
     default: "false"
     required: false
+  pr-title:
+    description: "Custom pull request title"
+    required: false
```

Refactor (blocking): Set the default PR title here.

```suggestion
  pr-title:
    description: "Custom pull request title"
    default: "chore (automated): OpenSauced updates"
    required: false
```